src/Core/Services/Implementations/UserService.cs

```diff
@@ -564,7 +564,9 @@ public async Task InitiateEmailChangeAsync(User user, string newEmail)
         user.Key = key;
         user.Email = newEmail;
         user.EmailVerified = true;
-        user.RevisionDate = user.AccountRevisionDate = DateTime.UtcNow;
+        var now = DateTime.UtcNow;
```

Comment:
ℹ️ Minor, but this date capture could move up a bit to higher in the method in all cases.